taca/utils/bioinfo_tab.py

```diff
@@ -60,12 +60,7 @@ def update_statusdb(run_dir):
     view = db.view('latest_data/sample_id')
     #Construction and sending of individual records, if samplesheet is incorrectly formatted the loop is skipped
     for flowcell in project_info:
-        if flowcell == 'UNKNOWN':
-            obj={'run_id':run_id, 'project_id':'ERROR_Samplesheet'}
-            logger.info("INVALID SAMPLESHEET, CHECK {} FORMED AT {}".format(run_id, valueskey))
-            error_emailer('no_samplesheet', run_id)
-            db.save(obj)
-        else:
+        if not flowcell == 'UNKNOWN':
```

Comment:
This is true every time `get_ss_projects ` returns `UNKNOWN`, the function parse the samplesheet and tries to find project name and sample name and other things...
It returns `UNKNOWN` when the sample sheet is not found .... from the mails I see a lot of problems...
 - we get mail about FC i cannot find anywhere: `160412_M01320_0053_000000000-AMRYR`
 - we get mails about FC in `/srv/illumina/miseq/IGNORE/` which are FCs that needs to... be ignored

seems like we get a mail for all MiSeq FCs.... the reg expression inline 154 fails continuosly but I do not get why.....  

Anyway, the cases in MiSeq are the following:
  -SampleSheet.csv is either located in Data/intensities..... or in the main run dir folder (you handle this)
  - if no samplesheet is present in these two location a warning is needed
  - we are interested only in MiSeq runs that have value `Production` of `Application` in the field `Description` in the `[Header]` section. See https://github.com/SciLifeLab/TACA/blob/master/taca/illumina/MiSeq_Runs.py#L33
  - we ignore everything else, i.e., MiSeq runs with samplesheet and Description different from Production and Application